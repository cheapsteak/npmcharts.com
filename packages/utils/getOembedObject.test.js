const getOembedObject = require('./getOembedObject');

test('getOembedObject output matches snapshot', () => {
  expect(
    getOembedObject({
      url: '/',
      width: 700,
      height: 500,
    }),
  ).toMatchSnapshot();

  expect(
    getOembedObject({
      url:
        '/compare/glamor,aphrodite,radium,glamorous,styled-components,jss,emotion',
      width: 700,
      height: 500,
    }),
  ).toMatchSnapshot();
});
