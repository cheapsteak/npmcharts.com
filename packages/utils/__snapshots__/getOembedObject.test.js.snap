// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getOembedObject output matches snapshot 1`] = `
Object {
  "height": 500,
  "html": "<iframe style=\\"width: 100%; overflow: hidden;\\" src=\\"https://null/?minimal=true\\" width=\\"700\\" height=\\"500\\" frameborder=\\"0\\" scrolling=\\"no\\" ></iframe>",
  "provider_name": "npmcharts",
  "provider_url": "https://npmcharts.com",
  "title": "Compare download trends of npm packages - npmcharts ðŸ“ˆ",
  "type": "rich",
  "version": "1.0",
  "width": 700,
}
`;

exports[`getOembedObject output matches snapshot 2`] = `
Object {
  "height": 500,
  "html": "<iframe style=\\"width: 100%; overflow: hidden;\\" src=\\"https://null/compare/glamor,aphrodite,radium,glamorous,styled-components,jss,emotion?minimal=true\\" width=\\"700\\" height=\\"500\\" frameborder=\\"0\\" scrolling=\\"no\\" ></iframe>",
  "provider_name": "npmcharts",
  "provider_url": "https://npmcharts.com",
  "title": "Compare npm downloads for glamor, aphrodite, radium, glamorous, styled-components, jss and emotion - npmcharts ðŸ“ˆ",
  "type": "rich",
  "version": "1.0",
  "width": 700,
}
`;
